README.md

```diff
@@ -75,16 +77,17 @@ GCC 4.9 / Clang 3.2 or later is recommended because OpenSpades relies on C++11 f
    cmake .. -DCMAKE_BUILD_TYPE=RelWithDebInfo && make
    ```
 
-4. Install OpenSpades (optional but recommended):
-
-   `sudo make install`
-
-   **note**: If you have a previous installation of OpenSpades, you have to uninstall it manually by `sudo rm -rf /usr/local/share/games/openspades` before installing a new one.
-
-5. Launch:
-
-   `openspades` (if installed) or `cd $REPO_DIRECTORY/openspades.mk; bin/OpenSpades` and enjoy
-
+4. Launching the game (follow the steps either A or B)
```

Comment:
IIRC `make install` doesn't launch the game.

Reply:
Ok, I added it as separate final step for each installation route